packages/harmony/.storybook/manager-head.html

```diff
@@ -0,0 +1,38 @@
+<style>
+  .sidebar-header > div {
+    margin-right: 0
+  }
+  .sidebar-header button[aria-label="Shortcuts"] {
+    display: none
+  }
+
+  .sidebar-subheading button[data-action="collapse-root"] {
+    display: flex;
+    padding: 4px 16px;
+    align-items: center;
+    gap: 8px;
+    border-radius: var(--border-radius-2-xl, 320px);
+    background: var(--violet-v-200, #C7BFFD);
```

Comment:
We generally don't keep the fallback values. Are they needed here?

Reply:
removed!